src/main/data/territories.json

```diff
@@ -307,7 +307,7 @@
   },
   {
     "dcncTag": "KP",
-    "dcncTerritory": "Democratic People's Republic of Korea",
+    "dcncTerritory": "North Korea",
```

Comment:
@SteveLLamb Do you mean that _North Korea_ matches current practice?

Reply:
> @SteveLLamb Do you mean that _North Korea_ matches current practice?



> @SteveLLamb Do you mean that _North Korea_ matches current practice?

No, we have used "South Korea" traditionally in Cinema instead of m49 display of "Republic of Korea".